src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -282,6 +301,18 @@ interface IOptionSettlementEngine {
      */
     function claim(uint256 tokenId) external view returns (Claim memory claimInfo);
 
+    /**
+     * @notice Returns the total amount of options written an exercised for all claims /
```

Comment:
Nice, this is super helpful doc. Nit — should it read "options written **and** exercised for all claims" ?

Reply:
yup good catch